examples/http/client.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+require 'rubygems'
+require 'bundler/setup'
+require 'faraday'
+# Require otel-ruby
+require 'opentelemetry'
```

Comment:
You shouldn't need to `require` this and the SDK -- just the SDK.

Reply:
@fbogsany okay, force of habit!